rootfs/api/models/build.py

```diff
@@ -43,10 +43,11 @@ def create(self, user, *args, **kwargs):
         try:
             self.app.deploy(new_release)
             return new_release
-        except Exception:
+        except Exception as e:
             if 'new_release' in locals():
                 new_release.delete()
-            raise
+
+            raise DeisException(str(e)) from e
```

Comment:
You'd get the error message and the status code, but not the parent Exception class. We should handle those on a case-by-case basis moving forward rather than mask them all off with a generic Exception class, IMO.